### PR TITLE
cid redirects added

### PR DESCRIPTION
**Pull request by JV0812**
cid redirect links are added since we have changed the path of classic and new dashboards.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

File: cid-redirects.json [hunk 1]
```diff
@@ -2641,8 +2641,83 @@
...
+  "/docs/dashboards/get-started/dashboard-optimization": "/docs/dashboards-classic/get-started/dashboard-optimization",
+  "/docs/dashboards/get-started/delete-dashboard": "/docs/dashboards-classic/get-started/delete-dashboard",
+  "/docs/dashboards/get-started/markdown-syntax": "/docs/dashboards-classic/get-started/markdown-syntax",
+  "/docs/dashboards/get-started/move-panel-dashboard": "/docs/dashboards-classic/get-started/move-panel-dashboard",
+  "/docs/dashboards/get-started/pause-panel": "/docs/dashboards-classic/get-started/pause-panel",
+  "/docs/dashboards/get-started/resize-panel": "/docs/dashboards-classic/get-started/resize-panel",
+  "/docs/dashboards/get-started/set-data-access-level-dashboard": "/docs/dashboards-classic/get-started/set-data-access-level-dashboard",
+  "/docs/dashboards/get-started/title-panels": "/docs/dashboards-classic/get-started/title-panels",
+  "/docs/dashboards/get-started/zoom-panel": "/docs/dashboards-classic/get-started/zoom-panel",
+  "/docs/dashboards-classic/chart-panel-types": "/docs/dashboards-classic/chart-panel-types",
```
Comment (kimsauce) [inline cid-redirects.json:2701]: @JV0812 it looks like the redirects from this line down are all linking to themselves. Once you fix these, it should work.

**Review by kimsauce — CHANGES_REQUESTED** [review]
see comment

File: cid-redirects.json (hunk 1 above)
Comment (JV0812) [reply cid-redirects.json]: Thanks Kim! I have updated those.

**Review by kimsauce — APPROVED** [review]

File: cid-redirects.json (hunk 1 above)
Comment (kimsauce) [reply cid-redirects.json]: thank you!

File: cid-redirects.json [hunk 2]
```diff
@@ -2641,8 +2641,83 @@
...
+  "/docs/dashboards-new/panels/markdown-syntax": "/docs/dashboards/panels/markdown-syntax",
+  "/docs/dashboards-new/panels/pie-charts": "/docs/dashboards/panels/pie-charts",
+  "/docs/dashboards-new/panels/sankey-charts": "/docs/dashboards/panels/sankey-charts",
+  "/docs/dashboards-new/panels/scatter-charts": "/docs/dashboards/panels/scatter-charts",
+  "/docs/dashboards-new/panels/single-value-charts": "/docs/dashboards/panels/single-value-charts",
+  "/docs/dashboards-new/panels/table-charts": "/docs/dashboards/panels/table-charts",
+  "/docs/dashboards": "/docs/dashboards-classic",
+  "/docs/dashboards/about": "/docs/dashboards-classic/about",
+  "/docs/dashboards/share-dashboard-inside-org": "/docs/dashboards-classic/share-dashboard-inside-org",
+  "/docs/dashboards/share-dashboard-outside-org": "/docs/dashboards-classic/share-dashboard-outside-org",
```
Comment (kimsauce) [inline cid-redirects.json:2684]: @JV0812 does this line need to be fixed?

Comment (JV0812) [reply cid-redirects.json:2684]: @kimsauce We don't want to change anything here.

File: sidebars.ts [hunk 3]
```diff
@@ -889,7 +889,7 @@ module.exports = {
       collapsed: true,
       link: {type: 'doc', id: 'dashboards/index'},
       items: [
-        'dashboards/about',
+        'dashboards/about-dashboard',
```
Comment (kimsauce) [inline sidebars.ts:892]:
```suggestion
        'dashboards/about',
```

File: sidebars.ts [hunk 4]
```diff
@@ -924,7 +924,7 @@ module.exports = {
         'dashboards/link-dashboards',
         'dashboards/set-data-access-level',
         'dashboards/share-dashboard-new',
-        'dashboards/share-dashboard-outside-org',
+        'dashboards/share-dashboard-outside',
```
Comment (kimsauce) [inline sidebars.ts:927]:
```suggestion
        'dashboards/share-dashboard-outside-org',
```

File: docs/dashboards/share-dashboards-outside.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-id: share-dashboard-outside-org
+id: share-dashboard-outside
```
Comment (kimsauce) [inline docs/dashboards/share-dashboards-outside.md:2]:
```suggestion
id: share-dashboard-outside-org
```

File: cid-redirects.json [hunk 6]
```diff
@@ -1375,8 +1375,8 @@
   "/docs/api/tokens": "/docs/api/token-management",
   "/docs/api/roles": "/docs/api/role-management",
   "/docs/api/users": "/docs/api/user-management",
-  "/Dashboards-and-Alerts/Dashboards": "/docs/dashboards",
-  "/Dashboards-and-Alerts/Dashboards/01-About-Dashboards": "/docs/dashboards/about",
+  "/Dashboards-and-Alerts/Dashboards": "/docs/dashboards-new",
+  "/Dashboards-and-Alerts/Dashboards/01-About-Dashboards": "/docs/dashboards/about-dashboard",
```
Comment (kimsauce) [inline cid-redirects.json:1379]:
```suggestion
  "/Dashboards-and-Alerts/Dashboards/01-About-Dashboards": "/docs/dashboards/about",
```

File: cid-redirects.json [hunk 7]
```diff
@@ -1635,7 +1635,7 @@
   "/cid/10230": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-webtx-source",
   "/cid/1092": "/docs/search/search-query-language/group-aggregate-operators/stddev",
   "/cid/1093": "/docs/metrics/metric-charts/log-overlay-analyze-metrics-visualizations",
-  "/cid/1094": "/docs/dashboards/share-dashboard-outside-org",
+  "/cid/1094": "/docs/dashboards/share-dashboard-outside",
```
Comment (kimsauce) [inline cid-redirects.json:1638]:
```suggestion
  "/cid/1094": "/docs/dashboards/share-dashboard-outside-org",
```

File: cid-redirects.json [hunk 8]
```diff
@@ -2248,7 +2248,7 @@
   "/cid/5421": "/docs/search/search-query-language/search-operators/fillmissing",
   "/cid/5422": "/docs/search/time-compare",
   "/cid/5423": "/docs/send-data/installed-collectors/collector-installation-reference/force-collectors-name-clobber",
-  "/cid/5424": "/docs/dashboards/about",
+  "/cid/5424": "/docs/dashboards/about-dashboard",
```
Comment (kimsauce) [inline cid-redirects.json:2251]:
```suggestion
  "/cid/5424": "/docs/dashboards/about",
```

File: docs/dashboards/about-dashboard.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-id: about
+id: about-dashboard
```
Comment (kimsauce) [inline docs/dashboards/about-dashboard.md:2]:
```suggestion
id: about
```

File: docs/dashboards/index.md [hunk 10]
```diff
@@ -92,7 +92,7 @@ In this section, we'll introduce the following concepts:
 </div>
 <div className="box smallbox14 card">
   <div className="container">
-  <a href="/docs/dashboards/share-dashboard-outside-org"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>Sharing Dashboards Outside Your Organization</h4></a>
+  <a href="/docs/dashboards/share-dashboard-outside"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>Sharing Dashboards Outside Your Organization</h4></a>
```
Comment (kimsauce) [inline docs/dashboards/index.md:95]:
```suggestion
  <a href="/docs/dashboards/share-dashboard-outside-org"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>Sharing Dashboards Outside Your Organization</h4></a>
```

File: docs/dashboards/index.md [hunk 11]
```diff
@@ -14,7 +14,7 @@ In this section, we'll introduce the following concepts:
 <div className="box-wrapper" markdown="1">
 <div className="box smallbox1 card">
   <div className="container">
-  <a href="/docs/dashboards/about"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>About Dashboards </h4></a>
+  <a href="/docs/dashboards/about-dashboard"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>About Dashboards </h4></a>
```
Comment (kimsauce) [inline docs/dashboards/index.md:17]:
```suggestion
  <a href="/docs/dashboards/about"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>About Dashboards </h4></a>
```

File: docs/dashboards/index.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
-slug: /dashboards
+slug: /dashboards-new
```
Comment (kimsauce) [inline docs/dashboards/index.md:2]:
```suggestion
slug: /dashboards
```

File: cid-redirects.json [hunk 13]
```diff
@@ -2641,8 +2641,82 @@
...
+  "/docs/dashboards-new/explore-view": "/docs/dashboards/explore-view",
+  "/docs/dashboards-new/drill-down-to-discover-root-causes": "/docs/dashboards/drill-down-to-discover-root-causes",
+  "/docs/dashboards-new/locate-deviations-time-series": "/docs/dashboards/locate-deviations-time-series",
+  "/docs/dashboards-new/set-custom-time-ranges": "/docs/dashboards/set-custom-time-ranges",
+  "/docs/dashboards-new/scheduled-report": "/docs/dashboards/scheduled-report",
+  "/docs/dashboards-new/export-dashboard-new": "/docs/dashboards/export-dashboard-new",
+  "/docs/dashboards-new/link-dashboards": "/docs/dashboards/link-dashboards",
+  "/docs/dashboards-new/set-data-access-level": "/docs/dashboards/set-data-access-level",
+  "/docs/dashboards-new/share-dashboard-new": "/docs/dashboards/share-dashboard-new",
+  "/docs/dashboards-new/share-dashboard-outside-org": "/docs/dashboards/share-dashboard-outside",
```
Comment (kimsauce) [inline cid-redirects.json:2657]:
```suggestion
  "/docs/dashboards-new/share-dashboard-outside-org": "/docs/dashboards/share-dashboard-outside-org",
```

File: cid-redirects.json [hunk 14]
```diff
@@ -2641,8 +2641,82 @@
...
+  "/docs/dashboards/chart-panel-types/numerical-single-value-charts": "/docs/dashboards-classic/chart-panel-types/numerical-single-value-charts",
+  "/docs/dashboards/chart-panel-types/string-single-value-charts": "/docs/dashboards-classic/chart-panel-types/string-single-value-charts",
+  "/docs/dashboards/chart-panel-types/table-charts": "/docs/dashboards-classic/chart-panel-types/table-charts",
+  "/docs/dashboards/edit-dashboards": "/docs/dashboards-classic/edit-dashboards",
+  "/docs/dashboards/edit-dashboards/manage-dashboards": "/docs/dashboards-classic/edit-dashboards/manage-dashboards",
+  "/docs/dashboards/edit-dashboards/manage-charts": "/docs/dashboards-classic/edit-dashboards/manage-charts",
+  "/docs/dashboards/edit-dashboards/manage-panels": "/docs/dashboards-classic/edit-dashboards/manage-panels",
+  "/docs/dashboards/edit-dashboards/manage-axis": "/docs/dashboards-classic/edit-dashboards/manage-axis",
+  "/docs/dashboards/edit-dashboards/set-time-range-dashboards": "/docs/dashboards-classic/edit-dashboards/set-time-range-dashboards",
+  "/docs/dashboards/edit-dashboards/use-filters-dashboards": "/docs/dashboards-classic/edit-dashboards/use-filters-dashboards",
```
Comment (kimsauce) [inline cid-redirects.json:2719]:
```suggestion
```

**Pull request merged**